src/app/components/EventsHighlight.tsx

```diff
@@ -1,6 +1,184 @@
-import { Box } from "@mui/material";
-import React from "react";
+"use client";
+import { Box, Typography } from "@mui/material";
+import React, { useState, useEffect } from "react";
+import Image from "next/image";
+import highlightBoxStyle from "./HighlightStyle";
+
+
+export const EventsHighlight = () => {
+  const imgs = [
+    "/images/EventsHighlights/home2.png",
+    "/images/EventsHighlights/search.png",
+    "/images/EventsHighlights/icons8-marketplace-48.png",
+    "/images/EventsHighlights/RAIN.png",
+    "/images/EventsHighlights/menu.png" ,
+  ];
+  const eventImages = [
+    "/images/EventsHighlights/home2.png",
+    "/images/EventsHighlights/search.png",
+    "/images/EventsHighlights/icons8-marketplace-48.png",
+    "/images/EventsHighlights/RAIN.png",
+    "/images/EventsHighlights/menu.png" ,
+  ];
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCurrentIndex((prevIndex) => (prevIndex + 1) % imgs.length);
+    }, 3000); //  3 seconds
+
+    return () => clearInterval(interval);
+  }, [imgs.length]);
+
+  return (
+    <Box
+      sx={{
+        height: "150vh",
+        width: "100vw",
+        backgroundColor: "#0E1322",
+        marginTop: "10vh",
+        display: "flex-col",
+      }}
+    >
+      {/* for "event highlight" */}
+      <Box sx={{ width: "100%", height: "10vh", display: "flex" }}>
+        <Typography
+          sx={{
+            margin: "auto",
+            fontSize: "7vh",
+            fontWeight: "400",
+            fontFamily: "",
+          }}
+        >
+          Event Highlights
+        </Typography>
```

Comment:
for fonts use variant in Typography so that fontsize will be consistent
```suggestion
        <Typography
        variant="h1"
          sx={{
            margin: "auto",
            fontWeight: "400",
            fontFamily: "",
          }}
        >
          Event Highlights
        </Typography>
```